client/webpack.client.rails.build.config.js

```diff
@@ -69,6 +69,9 @@ if (devBuild) {
   console.log('Webpack dev build for Rails'); // eslint-disable-line no-console
   config.devtool = 'eval-source-map';
 } else {
+  config.plugins.push(
+    new webpack.optimize.UglifyJsPlugin()
+  );
```

Comment:
It's absolutely needed for client bundles w/o assets pipeline, but I remember I've seen somewhere that re-parsing of minified builds might increase build time (I might mix up something), so I'm not sure about it w/ pipeline. For server side it can slightly improve performance, never measured it though.